packages/amplify_api_plugin_interface/lib/src/GraphQLResponse.dart

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:flutter/foundation.dart';
+import './types.dart';
+
+class GraphQLResponse<T> {
+  T data;
+  List<GraphQLResponseError> errors = [];
+
+  GraphQLResponse({@required this.data, this.errors});
+}
```

Comment:
We don't need to support GraphQL [extensions](https://dgraph.io/docs/graphql/api/requests/#extensions-field)?

Reply:
For reference, we don't support extensions on a response for JS, iOS or Android today, and there hasn't been a use case for it yet.

Reply:
As Richard mentioned, its not supported by Amplify libraries at this time.